user-manual/transfer/transfer.rst

```diff
@@ -54,6 +54,9 @@ transfer(s). Once this is done, the SIP can be moved into ingest or sent to a
 
    If your transfer is composed of digital forensic disk images, please see
    :ref:`Forensic image processing <forensic>`.
+   
+   If your transfer is a Dataverse dataset, please see 
+   :ref:`Dataverse Integration <dataverse>`.
```

Comment:
Dataset seems a little diminutive, although I appreciate that is technically what we're requesting from the API. We've seen fairly significant transfers from just our demo site. 

I would suggest the text:

`If your transfer is from a Dataverse repository, please see ...`

Reply:
that's interesting... I actually think of many datasets being massive (e.g. astronomy examples I've heard of that are TB big). But your wording suggestion works just as well so have updated to that.

Reply:
Good point! One of the first presentations I saw in this field was CERN. They had a little bit of data I suppose ;)